tools/aliaslookup/aliaslookup.py

```diff
@@ -22,20 +22,21 @@ def lookup_by_aliases(client: datastore.Client,
   Returns:
     a datastore.query.Iterator
   """
```

Comment:
nit: This doesn't seem right. I think it yields entities?
I think it was also wrong before the change?

Reply:
Agreed, thanks. Fixed in 65e80f09